src/lib/flow/sol-flow-resolver.c

```diff
@@ -105,9 +107,16 @@ sol_flow_resolve(
 
     err = resolver->resolve(resolver->data, id, &tmp_type, &tmp_named_opts);
     if (err < 0) {
-        SOL_DBG("could not resolve module for id='%s' using resolver=%s",
-            id, resolver->name);
-        return -ENOENT;
+        type_name = sol_conffile_resolve_alias(sol_str_slice_from_str(id));
+        if (!type_name)
+            return -ENOENT;
+        err = resolver->resolve(resolver->data, type_name, &tmp_type,
+            &tmp_named_opts);
+        if (err < 0) {
+            SOL_DBG("could not resolve module for id='%s' using resolver=%s",
```

Comment:
improve error message by showing both the alias (id) and type_name, also if err is errno, use the strerror()